beginner_source/fgsm_tutorial.py

```diff
@@ -99,6 +99,11 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
+from six.moves import urllib
```

Comment:
We should make sure to add a note and a link to https://github.com/pytorch/vision/issues/3497

```suggestion
# NOTE: This is a hack to get around "User-agent" limitations when downloading MNIST datasets
#       see, https://github.com/pytorch/vision/issues/3497 for more information
from six.moves import urllib
```